src/components/CardView.js

```diff
@@ -34,14 +34,18 @@ class CardView extends Component {
             this.state.isFlipped ? "front-flipped" : ""
           } front`}
         />
-        <div className={`${this.state.isFlipped ? "back-flipped" : ""} back`} />
+        <div
+          className={`${this.state.isFlipped ? "back-flipped" : ""} back`}
+          style={{ backgroundImage: `url(${this.props.backgroundReverse})` }}
+        />
       </div>
     );
   }
 }
 
 CardView.propTypes = {
   background: PropTypes.string.isRequired,
+  backgroundReverse: PropTypes.string.isRequired,
```

Comment:
The names shouldbe a bit more descriptive on stuff that is not obvious and less on stuff that is e.g. `cardRevImgUrl` `cardImgUrl` (for the cards it's just an image, for CSS it's background but shouldn't matter here)